scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -436,77 +413,47 @@ public void onClick(MenuItemClickEvent event) {
         submitButton.setIcon(SchedulerImages.instance.job_submit_16().getSafeUri().asString());
         submitButton.setIconSize(20);
         submitButton.setTooltip("Submit a new job");
-        submitButton.addClickHandler(new ClickHandler() {
-            public void onClick(ClickEvent event) {
-                SubmitWindow w = new SubmitWindow(SchedulerPage.this.controller);
-                w.show();
-            }
+        submitButton.addClickHandler(event -> {
+            SubmitWindow w = new SubmitWindow();
+            w.show();
         });
 
         ToolStripButton planButton = new ToolStripButton("Plan job");
         planButton.setIcon(SchedulerImages.instance.job_plan_16().getSafeUri().asString());
         planButton.setIconSize(20);
         planButton.setTooltip("Plan a job");
-        planButton.addClickHandler(new ClickHandler() {
-            public void onClick(ClickEvent event) {
-                PlanWindow w = new PlanWindow(SchedulerPage.this.controller);
-                w.show();
-            }
+        planButton.addClickHandler(event -> {
+            PlanWindow w = new PlanWindow(SchedulerPage.this.controller);
+            w.show();
         });
 
         schedStartButton = new MenuItem(START);
         schedStartButton.setIcon(SchedulerImages.instance.scheduler_start_16().getSafeUri().asString());
-        schedStartButton.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-            public void onClick(MenuItemClickEvent event) {
-                SchedulerPage.this.controller.startScheduler();
-            }
-        });
+        schedStartButton.addClickHandler(event -> SchedulerPage.this.controller.startScheduler());
 
         schedStopButton = new MenuItem(STOP);
         schedStopButton.setIcon(SchedulerImages.instance.scheduler_stop_16().getSafeUri().asString());
-        schedStopButton.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-            public void onClick(MenuItemClickEvent event) {
-                SchedulerPage.this.controller.stopScheduler();
-            }
-        });
+        schedStopButton.addClickHandler(event -> SchedulerPage.this.controller.stopScheduler());
 
         schedFreezeButton = new MenuItem(FREEZE);
         schedFreezeButton.setIcon(SchedulerImages.instance.scheduler_freeze_16().getSafeUri().asString());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)